File: worker/repository/cliente.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const cliente = new mongoose.Schema({
    nome: {
        type: String,
        require: true
    },
    email: {
        type: String,
        require: true
    },
    telefone: {
        type: String,
        require: true
    }
})

module.exports = mongoose.model('Cliente', cliente)

File: clientes-service/src/main/java/br/com/grifoo/clientesservice/Cliente.java
package br.com.grifoo.clientesservice;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Document(collection = "clientes")
public class Cliente {
	@Id
	public String id;
	public String nome;
	public String email;
	public String telefone;
}
